mllib/src/test/scala/org/apache/spark/mllib/feature/Word2VecSuite.scala

```diff
@@ -108,5 +108,26 @@ class Word2VecSuite extends SparkFunSuite with MLlibTestSparkContext {
     }
   }
 
+  test("test Infinity similarity for word vector with large value") {
```

Comment:
this should probably read "test similarity for word vectors with large values is not Infinity or NaN" for clarity